fixtures/utils.js

```diff
@@ -1,11 +1,16 @@
-var uuid = require('node-uuid');
-var userIds = [
+'use strict';
+
+const uuid = require('node-uuid');
+const controllerUtils = require('../census/controllers/utils');
+
+const userIds = [
   uuid.v4(),
   uuid.v4(),
   uuid.v4(),
   uuid.v4()
 ];
 
 module.exports = {
-  userIds: userIds
+  userIds: userIds,
```

Comment:
You can use the new [object literals syntax](http://www.benmvp.com/learning-es6-enhanced-object-literals/):

```js
module.exports = {
  userIds,
  ANONYMOUS_USER_ID: controllerUtils.ANONYMOUS_USER_ID
}
```